public/pages/workflows/workflow_list/workflow_list.test.tsx

```diff
@@ -18,6 +22,33 @@ jest.mock('../../../services', () => {
   };
 });
 
+const workflows = new Array(20).fill(null).map((_, index) => ({
+  id: `workflow_id_${index}`,
+  name: `workflow_name_${index}`,
+  type: Object.values(WORKFLOW_TYPE)[
+    Math.floor(Math.random() * Object.values(WORKFLOW_TYPE).length)
+  ],
+}));
+const workflowSet: [
+  string,
+  string,
+  WORKFLOW_TYPE
+][] = workflows.map(({ id, name, type }): [string, string, WORKFLOW_TYPE] => [
+  id,
+  name,
+  type,
+]);
```

Comment:
I don't think this is needed? `workflows` is already producing an array of objs containing `id, name, type`. Also, I'm seeing the pattern `id, name, type` repeated several times, usually a good sign to have a constant. How about having a constant somewhere that can be referenced instead? something like:
```
type TestWorkflowInput {
  id: string,
  name: string,
  type: WORKFLOW_TYPE
}
```